### PR TITLE
Add sample data in case of first time startup

### LINKED ISSUE #107: Missing sample data upon startup
The application does not have any sample data for startup, and thus is unable to bootup if distributed as a JAR file.

### PR DESCRIPTION
**Pull request by gitsac**
Closes #107

### CODE REVIEW

**Comment by gitsac** [comment]
Closes #107

File: src/main/java/seedu/address/model/util/SampleExpenseTracker.java
Comment (nicleejy) [inline src/main/java/seedu/address/model/util/SampleExpenseTracker.java:1]: LGTM!

File: src/main/java/seedu/address/MainApp.java
Comment (nicleejy) [inline src/main/java/seedu/address/MainApp.java:1]: This change appropriately displays default expenses and categories for a new user. Looks good to me!

**Review by nicleejy — APPROVED** [review]

File: src/main/java/seedu/address/model/util/SampleExpenseTracker.java
Comment (randallnhr) [reply src/main/java/seedu/address/model/util/SampleExpenseTracker.java:1]: LGTM as well!

**Review by randallnhr — COMMENTED** [review]
LGTM!

**Pull request merged**